source/docs/articles/sites/code/sso-and-identity-federation.md

```diff
@@ -11,6 +11,11 @@ Pantheon’s flexible infrastructure does not restrict protocols or ports used f
 
 Use SSL certificates for encrypted secure communication with externally hosted servers for authentication.
 
+## Two-Factor Authentication
+Two-factor authentication (TFA) is a security practice that requires users of your website to provide, along with their standard username and password, an additional form of authentication to log in.
+
+We strongly recommend using SAML (both for sites and the Dashboard) and OneLogin. Learn more about [two-factor authentication](/docs/guides/two-factor-authentication/).
```

Comment:
We should strongly recommend using SAML but not OneLogin. There are multiple legitimate options for SAML IdP out there.